build.gradle

```diff
@@ -66,6 +67,8 @@ dependencies {
     androidTestCompile fileTree(include: '*.jar', dir: 'catroidTest/libs')
     androidTestCompile 'com.jayway.android.robotium:robotium-solo:5.2.1'
     pmd 'net.sourceforge.pmd:pmd:5.1.1'
+    compile files('catroid/libs/Firmata-2.6.jar')
+    compile files('catroid/libs/FirmataSerial-1.1.jar')
 }
```

Comment:
Remove those 2 lines. [Line:65](https://github.com/Catrobat/Catroid/pull/1266/files#diff-c197962302397baf3a4cc36463dce5eaR65) should already handle compilation of your libraries. :palm_tree: